PORTAL/jobs/_job_compile_bench.py

```diff
@@ -44,11 +45,8 @@ class CompileBenchRequest(Request):
     PYPERFORMANCE = _utils.GitHubTarget.from_origin('python', 'pyperformance')
     PYSTON_BENCHMARKS = _utils.GitHubTarget.from_origin('pyston', 'python-macrobenchmarks')
 
-    #pyperformance = PYPERFORMANCE.copy('034f58b')  # 1.0.4 release (2022-01-26)
```

Comment:
Consider leaving this or adding a comment to give an example of helpful info to report for the revision.

Reply:
Sure -- I can just put it back in.